Source/WebCore/rendering/RenderObject.h

```diff
@@ -241,6 +241,7 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr<RenderOb
         IsFragmentedFlow = 1 << 1,
         IsTextControl = 1 << 2,
         IsSVGBlock = 1 << 3,
+        IsViewTransitionContainer = 1 << 4,
```

Comment:
I don't quite understand how this flag is applied, I only see it applied on ::view-transition and ::view-transition-new, I'm not sure I understand the logic there.

Reply:
It should be applied on `::view-transition`, `::view-transition-group` and `::view-transition-image-pair`. All the view transition pseudo elements that are containers.

`::view-transition-new` and `::view-transition-old` should be the ones that don't have it.

Reply:
Oh I misread the code, thanks